package.json

```diff
@@ -40,6 +40,8 @@
     "prefeature-tests": "gulp",
     "prepublish": "gulp",
     "test": "npm run unit-tests && npm run feature-tests",
-    "unit-tests": "./node_modules/.bin/mycha run --reporter spec test/spec"
+    "unit-tests": "./node_modules/.bin/mycha run --reporter spec test/spec",
+    "update": "npm-check-updates -u && rm -rf node_modules && npm install && npm test",
+    "update-check": "npm-check-updates"
```

Comment:
I find [david](https://www.npmjs.com/package/david) has a much cleaner and readable output as its in a table. The output of checking for updates also includes a command you can run to update specific packages. For open source repos you also get a badge